backend-lib/Cargo.toml

```diff
@@ -16,17 +16,18 @@ hdwallet = "0.4"
 hdwallet-bitcoin = "0.4"
 hex = "0.4"
 jni = { version = "0.21", default-features = false }
+orchard = "0.8"
 prost = "0.12"
 rusqlite = "0.29"
 sapling = { package = "sapling-crypto", version = "0.1", default-features = false }
 schemer = "0.2"
 secp256k1 = "0.26"
 secrecy = "0.8"
 zcash_address = "0.3"
-zcash_client_backend = { version = "0.11", features = ["transparent-inputs", "unstable"] }
-zcash_client_sqlite = { version = "0.9", features = ["transparent-inputs", "unstable"] }
-zcash_primitives = "0.14"
-zcash_proofs = "0.14"
+zcash_client_backend = { version = "0.12.1", features = ["orchard", "transparent-inputs", "unstable"] }
+zcash_client_sqlite = { version = "0.10.2", features = ["orchard", "transparent-inputs", "unstable"] }
```

Comment:
This needs the 0.10.3 upgrade: https://github.com/zcash/librustzcash/pull/1342

Reply:
#1428 merged. Thank you.